runtime/jvmti/jvmtiHelpers.c

```diff
@@ -855,28 +855,28 @@ getVirtualThreadState(J9VMThread *currentThread, jthread thread)
 				break;
 			}
 			case JVMTI_VTHREAD_STATE_RUNNABLE:
-				rc = JVMTI_JAVA_LANG_THREAD_STATE_RUNNABLE;
-				break;
-			case JVMTI_VTHREAD_STATE_RUNNABLE_SUSPENDED:
-				rc = JVMTI_JAVA_LANG_THREAD_STATE_RUNNABLE | JVMTI_THREAD_STATE_SUSPENDED;
-				break;
+				/* Fall Through */
 			case JVMTI_VTHREAD_STATE_RUNNING:
-				rc = JVMTI_JAVA_LANG_THREAD_STATE_RUNNABLE;
-				break;
+				/* Fall Through */
 			case JVMTI_VTHREAD_STATE_PARKING:
 				/* Fall Through */
+			case JVMTI_VTHREAD_STATE_TIMED_PARKING:
+				/* Fall Through */
```

Comment:
I would recommend against these "fall through" comments when there's no code between case labels. It's even given as a good example in https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-switch-statements.

Reply:
Addressed in https://github.com/eclipse-openj9/openj9/pull/18442.